built_value_generator/lib/src/value_source_class.dart

```diff
@@ -99,7 +108,25 @@ abstract class ValueSourceClass
   @memoized
   bool get hasPartStatement {
     final expectedCode = partStatement;
-    return element.library.source.contents.data.contains(expectedCode);
+    return source.contains(expectedCode);
+  }
+
+  @memoized
+  bool get hasBuiltValueImportWithShow {
+    // It would be more accurate to check using the AST, but this is
+    // potentially expensive. We already have the source for the "part of"
+    // check, use that.
+    return source.contains('$_importWithSingleQuotes show') ||
+        source.contains('$_importWithSingleQuotes show');
```

Comment:
_importWithDoubleQuotes

Reply:
Thanks! I already found and fixed that on a different revision at least once. Now added a test for it like I should have done in the first place...